zallet/src/components/json_rpc/methods/list_unspent.rs

```diff
@@ -97,7 +137,15 @@ pub(crate) fn call(wallet: &DbConnection) -> Response {
         None => return Ok(ResultType(vec![])),
     };
 
-    let mut unspent_notes = vec![];
+    let target_height = as_of_height.map_or(anchor_height, |h| {
+        if h > 0 {
+            anchor_height.saturating_sub(u32::try_from(h).unwrap())
+        } else {
+            anchor_height + 1
+        }
+    });
```

Comment:
It says at https://github.com/zcash/wallet/blob/69a035577780c8e2b0c9ef4b5bde341d32341b39/zallet/src/components/json_rpc/methods.rs#L273 that this parameter is not supported however there seems to be logic here for it.